Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Resources/devices.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "iPhone_NAME": {
+    "capabilities": {
+      "platformVersion": "<iOS_platform_version>",
+      "deviceName": "<iOS_device_name>",
+      "udid": "<iOS_device_udidi>",
+      "wdaLocalPort": "<wda_local_port>"
+    }
+  },
+  "Android_NAME": {
+    "capabilities": {
+      "deviceName": "<Android_device_name>",
+      "udid": "<Android_device_udid>",
+      "systemPort": "<appium_system_port>"
+    }
+  }
```

Comment:
can we have emulator and simulator capability examples anywhere?

Reply:
Do we really need it? The difference is only in capabilities. Btw, we can set simulator/enualtor as a default device.

Reply:
I've add examples on simulator/emulator settings and set them as default devices.